[Bug]:  BABEL_3_1_STABLE babelfish_extensions/contrib/babelfishpg_tds to be unable to build

### What happened?

When I execute (BABEL_3_1_STABLE) make on ubuntu 20.04, the compilation reports an error. 

```
 cd ${PG_SRC}/contrib/babelfishpg_tsql
 make -j ${JOBS}
```

### Version

BABEL_3_X_DEV (Default)

### Extension

babelfishpg_tds

### Which flavor of Linux are you using when you see the bug?

Ubuntu (Default)

### Relevant log output

```shell
In file included from src/linked_servers.c:11:
src/linked_servers.h:2:10: fatal error: sybdb.h: No such file or directory
    2 | #include "sybdb.h"
      |          ^~~~~~~~~
compilation terminated.
```


### Code of Conduct

- [X] I agree to follow this project's Code of Conduct.

--- a/.github/composite-actions/build-extensions/action.yml
+++ b/.github/composite-actions/build-extensions/action.yml
@@ -36,5 +36,6 @@ runs:
         cd ../babelfishpg_tds
         make -j 4 && make install
         cd ../babelfishpg_tsql
-        make && make install
+        PG_CPPFLAGS='-I/usr/include -DENABLE_TDS_LIB' SHLIB_LINK='-lsybdb -L/usr/lib64' make
+        PG_CPPFLAGS='-I/usr/include -DENABLE_TDS_LIB' SHLIB_LINK='-lsybdb -L/usr/lib64' make install
       shell: bash

--- a/contrib/README.md
+++ b/contrib/README.md
@@ -219,3 +219,19 @@ The following build instructions comply with Ubuntu 20.04 and Amazon Linux 2 env
       mvn test
       ```
 For detailed instructions on how to write, add, and run tests in JDBC test framework, refer [to the online instructions](../test/JDBC/README.md).
+
+
+# How to build the babelfishpg_tsql extension with linked servers enabled
+
+1. To work with linked servers, you must install the `tds_fdw` extension. More information about building and installing the extension can be found [at this link](https://github.com/tds-fdw/tds_fdw/blob/master/README.md).
+2. Build the babelfishpg_tsql extension as follows:
+    ```
+    PG_CPPFLAGS='-I/usr/include -DENABLE_TDS_LIB' SHLIB_LINK='-lsybdb -L/usr/lib64' make
+    PG_CPPFLAGS='-I/usr/include -DENABLE_TDS_LIB' SHLIB_LINK='-lsybdb -L/usr/lib64' make install
+    ```
+3. Create rest of the Babelfish extensions as usual, and initialize Babelfish.
+4. Create the `tds_fdw` extension in the Babelfish database:
+    ```
+    babelfish_db=> CREATE EXTENSION tds_fdw;
+    CREATE EXTENSION
+    ```

--- a/contrib/babelfishpg_tsql/Makefile
+++ b/contrib/babelfishpg_tsql/Makefile
@@ -78,10 +78,6 @@ export ANTLR4_RUNTIME_LIB=-lantlr4-runtime
 export ANTLR4_RUNTIME_INCLUDE_DIR=/usr/local/include/antlr4-runtime
 export ANTLR4_RUNTIME_LIB_DIR=/usr/local/lib
 
-export FREE_TDS_INCLUDE_DIR=/usr/include
-export FREE_TDS_LIB_DIR=/usr/lib64
-export FREE_TDS_LIB="-lsybdb"
-
 OBJS += src/pltsql_bulkcopy.o
 
 PG_CXXFLAGS += -g -Werror
@@ -91,9 +87,9 @@ PG_CXXFLAGS += -Wno-register # otherwise C++17 gags on PostgreSQL headers
 PG_CXXFLAGS += -I$(ANTLR4_RUNTIME_INCLUDE_DIR)
 PG_CFLAGS += -g -Werror
 PG_CFLAGS += -fstack-protector-strong
-PG_CPPFLAGS += -I$(TSQLSRC) -I$(PG_SRC) -I$(FREE_TDS_INCLUDE_DIR) -DFAULT_INJECTOR -DENABLE_TDS_LIB
+PG_CPPFLAGS += -I$(TSQLSRC) -I$(PG_SRC) -DFAULT_INJECTOR
 
-SHLIB_LINK += -L$(ANTLR4_RUNTIME_LIB_DIR) $(ANTLR4_RUNTIME_LIB) -lcrypto $(FREE_TDS_LIB) -L$(FREE_TDS_LIB_DIR)
+SHLIB_LINK += -L$(ANTLR4_RUNTIME_LIB_DIR) $(ANTLR4_RUNTIME_LIB) -lcrypto
 
 UPGRADES = $(patsubst sql/upgrades/%.sql,sql/%.sql,$(wildcard sql/upgrades/*.sql))
 

